vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 
 cd /vagrant/
-#git reset --hard HEAD
-#./helper.sh --bumpmodules
+
+#install nvm
+curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```

Comment:
I'd also install node there

Reply:
I think it is installed by nvm? https://github.com/nvm-sh/nvm/blob/v0.35.3/install.sh#L67

Reply:
Can you try running node -v after installing to check if it works?

Reply:
no, I have no free space. @LeonVreling can you try?
`git pull` (inside oms-docker)
`git checkout chore/various-improvements`
`vagrant up --provision`

Reply:
or whoever is free earlier between @maaikeheijdenrijk @mibaru-aegee

Reply:
anyway I went lower in the code and yes, it does install node automatically, I think we don't need to check it (what good would an nvm do without node? imho it's obvious it's installed)

Reply:
nice then

Reply:
v12.14.0 (leon jansen confirmed that)